app/assets/stylesheets/_custom.css.scss

```diff
@@ -50,3 +46,11 @@ img {
   @extend .btn-default;
 }
 
+span.glyphicon-collapse-down {
+  text-decoration: none;
+  color: $gray-light;
+}
+
+span.glyphicon-collapse-down:hover, span.glyphicon-collapse-down:active {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line